pages/ksphere/konvoy/1.5.0-beta/release-notes/index.md

```diff
@@ -15,6 +15,61 @@ enterprise: false
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
 
+### Version v1.5.0-beta.5 - Released 1 June 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+|**Minimum** | 1.15.4 |
+|**Maximum** | 1.17.x |
+|**Default** | 1.17.6 |
+
+#### Bug fixes
+
+- API: Don't print `indexType` in `cluster.yaml` file.
+- Terraform: allow users to specify terraform go template in the `extras/provisioner` directory.
+- CLI: Fix a bug where deploying `extras/kubernetes` resources required setting the `KUBECONFIG` environment variable.
+- CLI: Fix a bug where running `konvoy drain/cordon/uncordon` required setting the `KUBECONFIG` environment variable.
+
+#### Improvements
+
+-   Update default Kubernetes version to `v1.17.6`.
+-   Update default Containerd version to `v1.3.4`.
+-   API: New `cluster.yaml` option `spec.kubernetes.imageRepository` to allow for pulling control-plane images from an alternative repository instead of the default `k8s.gcr.io`.
```

Comment:
```suggestion
-   API: New `cluster.yaml` option `spec.kubernetes.imageRepository` allows pulling control-plane images from an alternative repository instead of the default `k8s.gcr.io`.
```